src/foam/u2/wizard/StepWizardletController.js

```diff
@@ -178,6 +216,30 @@ foam.CLASS({
         });
       },
     },
+    function countAvailableSections(wizardletIndex) {
+      return this.sectionAvailableSlots[wizardletIndex].reduce(
+        (total, sectionAvailable$) =>
+          sectionAvailable$.get() ? total + 1 : total,
+        0
+      );
+    },
+    function canSkipTo(wizardletIndex) {
+      let currentWizardletIndex =
+        this.screenIndexToSection(this.subStack.pos)[0];
+      for ( let w = currentWizardletIndex ; w <= wizardletIndex ; w++ ) {
+        if ( ! this.wizardlets[w].readyToSubmit() ) return false;
+      }
+      return true;
+    },
+    function skipTo(screenIndex) {
+      var f;
```

Comment:
a function that returns a function? Where is this used? and why is this formatting necessary? If this isn't just an oversight, please document.

Reply:
`return f()` - it's not returning a function, it's a recursive call (needs to be because of the promise). I'll add a comment